docs/develop/api-reference/sorted-set-collections.md

```diff
@@ -0,0 +1,177 @@
+      * Miss
+* Cache miss (if the sorted set does not exist)
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### SortedSetRemove
+
+Removes an element from a sorted set.
+
+| Name            | Type             | Description                                   |
+| --------------- | ---------------- | --------------------------------------------- |
+| cacheName       | String           | Name of the cache.                            |
+| setName         | String           | Name of the set item to be altered. |
+| elements        | All \| string[] \| bytes[] | Element to be removed by this operation. |
+
+You can remove either elements all elements or a set of elements.
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### SortedSetGetRank
+
+What position is this element in the specificed sorted set?
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| cacheName       | String          | Name of the cache.                            |
+| setName         | String          | Name of the set item to be altered. |
+| element         | String \| bytes | Element to be removed by this operation. |
+
+<details>
+  <summary>Method response object</summary>
+
+* Hit
+  * Rank: integer
+* Miss
+* Error
+
+See [response objects](./response-objects.md) for specific information.
+
+</details>
+
+### SortedSetIncrementScore
+
+Adds to the score of an element. If the value is *missing* from the sorted set, this method sets the value to the amount to increment by.
+
+:::note
+
+The resulting incremented score must be between -9223372036854775808 and 9223372036854775807, ie. a signed double 64-bit float. If not, there will be an error response.
+
+:::
+
+Examples:
+
+- When the element does not exist in the sorted set, `SortedSetIncrementScore(cache, cacheName, setName, value, 10)` will set the element's score to 10.
+- When the existing element is a value:score of "{ 'KesselRun' : 12 }" , `SortedSetIncrementScore(cacheName, setName, value, 10)` will set the element's score to 22.
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| cacheName       | String          | Name of the cache.                            |
+| setName         | String          | Name of the sorted set item to be altered. |
+| value           | String \| bytes | Value for the element to be incremented by this operation. |
+| amount          | Number          | The quantity to add to the score. May be positive, negative, or zero. Defaults to 1. |          
+| ttl             | [CollectionTTL object](./collection-ttl.md) | This will come back as a TTL construct. |
```

Comment:
I'm thinking the description of the ttl should be consistent with what you already have on this page `TTL for the set item in cache. This TTL takes precedence over the TTL used when initializing a cache connection client.`